rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -13,6 +13,7 @@ resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/rapid7_insightvm_cloud
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
   vendor_url: https://www.rapid7.com
+  pair_scan_engine: https://docs.rapid7.com/insightvm/scan-engine-management-on-the-insight-platform/#how-to-pair-your-scan-engines-to-the-insight-platform
```

Comment:
Instead of adding here, we want this link to be added in help.md under the `References` section (different name than what extension library calls it). So I would remove this line here and then update References at the bottom of help.md with a link similar to what you have for the API documentation link